packages/fast-development-site-react/src/components/site/menu.tsx

```diff
@@ -25,6 +26,9 @@ const style: ComponentStyles<ISiteNavManagedClasses, IDevSiteDesignSystem> = {
     },
     siteMenu_nav: {
         position: "absolute",
+        background: "#FFFFFF",
+        zIndex: "5",
+        boxShadow: `outline ${toPx(-2)} 0 ${toPx(4)} ${toPx(4)} rgba(0, 0, 0, 0.2)`,
```

Comment:
We could use Chroma here to get the desired background boxshadow color.
`boxShadow: (config: IDesignSystem): string => {
      return `outline ${toPx(-2)} 0 ${toPx(4)} ${toPx(4)} ${Chroma(config.foregroundColor).alpha(0.2).css()}`;
  }`

Reply:
I think with our overall goal of removing Chroma where we can (because we aren't using fast-colors yet), we should hold on this. We have quite a few places we could use it here, but I think we intentionally left off the dependency. We definitely need a better way to manipulate the colors here as most all of them are hard coded or built into the design system for single use cases. I think for now though, we should hold on using Chroma here and come up with a viable solution.